documentation/User-Documentation/Commands-and-permissions.md

```diff
@@ -20,7 +20,7 @@
 * **/q delete {tag/point/objective/entry} {name}** - deletes all specified things in the database
 * **/q backup** - creates a backup of configuration files and database
 * **/q update** - updates the plugin to the newest version.
-* **/q create {package}**: creates new package with given name, filled with default quest
+* **/q create {package}**: creates new package with given name, and empty files
```

Comment:
```suggestion
* **/q create {package}**: creates new package with given name and empty files
```

Reply:
I thought over different possible formulations, but this is the one with the least impact.

Some other ideas:
```suggestion
* **/q create {package}**: creates new package with given name and fills it with empty files
```
```suggestion
* **/q create {package}**: creates new package with given name and creates empty quest files
```
```suggestion
* **/q create {package}**: creates new package with given name and creates empty files in it
```
```suggestion
* **/q create {package}**: creates new package with given name and creates empty files inside of it
```
Of course some combination of above are also possible and this is by no means an exhaustive list.